101-vm-automatic-static-ip/nested/update-nic.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Seems like we should use this twice for each time the nic is deployed - would have to pass in more params, but would make it more unique as a sample.  Not required, but something to think about...